tests/Microsoft.Identity.Test.Unit/BrokerTests/BrokerRequestTests.cs

```diff
@@ -34,7 +35,11 @@ namespace Microsoft.Identity.Test.Unit.BrokerTests
     [TestClass]
     [TestCategory("Broker")]
     public class BrokerTests : TestBase
-    {
+    {
+        const string TestErrCode = "TestErrCode";

```

Comment:
Wouldn't hurt to put these in the TestConstants.cs class to be reused elsewhere.